packages/vue-vanilla/src/controls/ControlWrapper.vue

```diff
@@ -78,9 +78,12 @@ export default defineComponent({
       return computeLabel(
         this.label,
         this.required,
-        !!this.appliedOptions?.hideRequiredAsterisk
+        true
       );
```

Comment:
handing over `true` here is the same as just using the `this.label` directly, so in fact we can remove the `computedLabel` completely once we manually manage the asterisk.